src/filefingerprint.cpp

```diff
@@ -165,10 +165,10 @@ bool FileFingerprint::genfingerprint(FileAccess* fa, bool ignoremtime)
             return true;
         }
 
-        for (unsigned i = 0; i < sizeof crc / sizeof *crc; i++)
+        for (unsigned i = 0; i < crc.size(); i++)
```

Comment:
Because `crc` is now a `std::array` we can get its size through the `.size()` member function